website/docs/security/authentication.md

```diff
@@ -103,3 +116,41 @@ If you encounter any issues while authenticating with FerretDB, try setting the
 Please refer to your specific driver documentation on how to set this field.
 
 If this does not resolve your issue please file a bug report [here](https://github.com/FerretDB/FerretDB/issues/new?assignees=ferretdb-bot&labels=code%2Fbug%2Cnot+ready&projects=&template=bug.yml).
+
+## Experimental authentication
+
+Upon enabling the flag `--test-enable-new-auth` or the environment variable `TEST_ENABLE_NEW_AUTH`,
+FerretDB enables user management and authentication against the stored credentials in the `admin` database `system.users` collection.
+The users are managed using the `usersInfo`, `createUser`, `updateUser`, `dropUser` and `dropAllUsersFromDatabase` commands.
+The users may be created and authenticated using `SCRAM-SHA-256`, `SCRAM-SHA-1` and `PLAIN` mechanisms.
+The `SCRAM` mechanisms are the algorithms used to store hashed passwords. FerretDB also stores hashed password using `PLAIN` mechanism.
+
+Please note that FerretDB does not support authorization yet, so created users currently have access to all databases.
```

Comment:
That's a strange place to say _again_ that authentication is not authorization. Saying it once at the begging is enough, I think.

Reply:
Removed this one.